contracts/payloads/IGP15/PayloadIGP15.sol

```diff
@@ -0,0 +1,668 @@
+        0x9D39A5DE30e57443BfF2A8307A4256c8797A3497;
+    address public constant USDC_ADDRESS =
+        0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48;
+    address public constant USDT_ADDRESS =
+        0xdAC17F958D2ee523a2206206994597C13D831ec7;
+
+    constructor() {
+        ADDRESS_THIS = address(this);
+    }
+
+    function propose(string memory description) external {
+        require(
+            msg.sender == PROPOSER ||
+                msg.sender == TEAM_MULTISIG ||
+                address(this) == PROPOSER_AVO_MULTISIG,
+            "msg.sender-not-allowed"
+        );
+
+        uint256 totalActions = 1;
+        address[] memory targets = new address[](totalActions);
+        uint256[] memory values = new uint256[](totalActions);
+        string[] memory signatures = new string[](totalActions);
+        bytes[] memory calldatas = new bytes[](totalActions);
+
+        // Action 1: call executePayload on timelock contract to execute payload related to Fluid
+        targets[0] = address(TIMELOCK);
+        values[0] = 0;
+        signatures[0] = "executePayload(address,string,bytes)";
+        calldatas[0] = abi.encode(ADDRESS_THIS, "execute()", abi.encode());
+
+        uint256 proposedId = GOVERNOR.propose(
+            targets,
+            values,
+            signatures,
+            calldatas,
+            description
+        );
+
+        require(proposedId == PROPOSAL_ID, "PROPOSAL_IS_NOT_SAME");
+    }
+
+    function execute() external {
+        require(address(this) == address(TIMELOCK), "not-valid-caller");
+
+        // Action 1: Set market rates for sUSDe on Liquidity.
+        action1();
+
+        // Action 2: Set token config for sUSDe on Liquidity.
+        action2();
+
+        // Action 3: Deploy sUSDe/USDC vault and set related configs.
+        action3();
+
+        // Action 4: Deploy sUSDe/USDT vault and set related configs.
+        action4();
+
+        // Action 5: Update market rates for USDC.
+        action5();
+
+        // Action 6: Update market rates for USDT.
+        action6();
+
+        // Action 7: Update reward rates for USDC.
+        action7();
+
+        // Action 8: Update reward rates for USDT.
+        action8();
+    }
+
+    function verifyProposal() external view {}
+
+    /***********************************|
+    |     Proposal Payload Actions      |
+    |__________________________________*/
+
+    /// @notice Action 1: Set market rates for sUSDe on Liquidity.
+    function action1() internal {
+        AdminModuleStructs.RateDataV2Params[]
+            memory params_ = new AdminModuleStructs.RateDataV2Params[](1);
+
+        params_[0] = AdminModuleStructs.RateDataV2Params({
+            token: sUSDe_ADDRESS, // sUSDe
+            kink1: 50 * 1e2, // 50%
+            kink2: 80 * 1e2, // 80%
+            rateAtUtilizationZero: 0, // 0%
+            rateAtUtilizationKink1: 20 * 1e2, // 20%
+            rateAtUtilizationKink2: 40 * 1e2, // 40%
```

Comment:
proposal says 10% for rateAtUtilizationKink1, then 20% and 50% here? https://atlas.instadapp.io/proposals/15

Reply:
Yes! Misplaced it. Although, doesn't matter as this curve is not being used.